components/Cast/Upvote.tsx

```diff
@@ -1,37 +1,25 @@
 import createReaction from '@/lib/neynar/createReaction';
-import getCastReactions from '@/lib/neynar/getCastReactions';
 import { useNeynarProvider } from '@/providers/NeynarProvider';
-import { useEffect, useState } from 'react';
+import { useState } from 'react';
 import { MdArrowCircleUp } from 'react-icons/md';
 import { Address } from 'viem';
 
-const Upvote = ({ target }: { target: Address }) => {
+const Upvote = ({ cast }: { cast: any }) => {
```

Comment:
can you use the same `CastType` from `components/Cast/Cast.tsx` instead of `any`?